build.gradle

```diff
@@ -275,22 +275,10 @@ task dockerBuild(type: Exec, dependsOn: shadowJar) {
 }
 
 task dockerBuildX(type: Exec, dependsOn: shadowJar) {
-    commandLine 'docker', 'buildx', 'build', '--load', '--platform', 'linux/arm64', '-t', dockerImageTag + '-arm64', project.rootDir
+    commandLine 'docker', 'buildx', 'build', '--push', '--platform', 'linux/arm64,linux/amd64,linux/arm/v7', '-t', dockerImageTag , project.rootDir
     doLast {
         exec {
-            commandLine 'docker', 'buildx', 'build', '--load', '--platform', 'linux/amd64', '-t', dockerImageTag + '-amd64', project.rootDir
-        }
-
-        exec {
-            commandLine 'docker', 'tag', dockerImageTag + '-arm64', "${dockerImageName}:latest-arm64-dev"
-        }
-
-        exec {
-            commandLine 'docker', 'tag', dockerImageTag + '-amd64', "${dockerImageName}:latest-amd64-dev"
-        }
-
-        exec {
-            commandLine 'docker', 'tag', dockerImageTag + '-amd64', "${dockerImageName}:latest-dev"
+            commandLine 'docker', 'buildx', 'build', '--push', '--platform', 'linux/arm64,linux/amd64,linux/arm/v7', '-t', "${dockerImageName}:latest-dev", project.rootDir
```

Comment:
For push you can use the `dockerPushX` task.

Reply:
I don't think I can. Because `docker buildx` builds the image within a container, the resulting image isn't available within the host OS in order to be able to push it, unless it is run with `docker buildx --output type=docker`, which isn't possible for multi-arch images. The push needs to be done with `docker buildx`.